FHSDK/Sync/FHSyncUtils.cs

```diff
@@ -75,7 +75,7 @@ private static JArray SortObj(object pObject)
 			} else if (jsonObj.Type == JTokenType.Object) {
 				JObject casted = jsonObj as JObject;
 				List<String> keys = casted.Properties ().Select (c => c.Name).ToList ();
-				keys.Sort ();
+				keys.Sort(Comparison);
```

Comment:
this is fine for English. But I am just a bit concerned if the customer has a dataset using some other languages.